gatsby-config.js

```diff
@@ -104,6 +108,28 @@ module.exports = {
             path: '/server/reference/'
           }
         ]
+      },
+      {
+        title: 'Recipes',
+        path: '/recipes/',
+        pages: [
+          {
+            title: 'Events',
+            path: '/recipes/events/'
+          },
+          {
+            title: 'Menus',
+            path: '/recipes/menus/'
+          },
+          {
+            title: 'Passing Arguments',
+            path: 'recipes/arguments/'
+          },
+          {
+            title: 'Setting Script Result',
+            path: 'recipes/script-result'
```

Comment:
Gatsby recommendation - start and end the path with '/'

Reply:
Done